src/utils/i18nTranslations.ts

```diff
@@ -34,28 +34,40 @@ export const TX_TYPE_TRANSLATIONS: TranslationMap<TxType> = {
   PayingForTx: () => tg('transaction.type.payingForTx'),
   SpendTx: () => tg('transaction.type.sentTx'),
 };
+export function getTxTypeLabel(txType: TxType): string {
+  const translateFunc = TX_TYPE_TRANSLATIONS[txType];
+  return translateFunc ? translateFunc() : '';
```

Comment:
```suggestion
  return TX_TYPE_TRANSLATIONS[txType] ? translateFunc() : '';
  
```

Reply:
This change makes no sense as you first suggest to remove the `translateFunc` constant and than you use it.